client/filestore-client/src/CordovaFileUpload.ts

```diff
@@ -0,0 +1,62 @@
+import * as Bluebird from 'bluebird';
+
+// Cordova mocks for typescript
+declare var FileUploadOptions: any;
+declare var FileTransfer: any;
```

Comment:
Maybe put these in private fields to a `CordovaFileUpload` class, or just rename the file so it doesn't suggest it's exporting a class.

Reply:
I had problem with this. There are definitions for this but they are outdated (more than year old) when bringing old definitions I could have some differences between the new ones
with such small exposure and couple trials I decided to go this way. 

I will wrap methods and fields into class.

Reply:
Ah, you mean ts definitions for the cordova plugins? Sorry, didn't know these were from cordova, thought they were new classes.

I was referring to the actual name of the file though, the module is exporting the individual functions, not a class, so I think we should avoid naming it starting with uppercase.

Reply:
They are globals from cordova libraries required in top level app. 
Sadly cordova requrires them to be on the top level application. as they are part of the build process.

TL;DR - what is your suggestion here :)

Reply:
So even if keeping the filetransfer plugin, what I found in other `@types/cordova-plugin-*` definitions is that they add the objects under `window.*` or `window.navigator.*`, and any browser global can be referenced like that too.